proposals/dao/repay_fuse_bad_debt.ts

```diff
@@ -0,0 +1,172 @@
+import { CToken, CTokenFuse, FuseFixer__factory } from '@custom-types/contracts';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner } from '@test/helpers';
+import { forceEth, forceEthMultiple, forceSpecificEth } from '@test/integration/setup/utils';
+import { utils } from 'ethers';
+import hre from 'hardhat';
+
+/*
+Withdraw FEI from Aave and Compound
+*/
+
+const fipNumber = 'repay_fuse_bad_debt'; // Change me!
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const fuseFixerFactory = (await hre.ethers.getContractFactory('FuseFixer')) as FuseFixer__factory;
+  const fuseFixer = await fuseFixerFactory.deploy(addresses.core);
```

Comment:
`.deploy()` just sends the `deploy` transaction. Good practice to wait for that to be mined with `await fuseFixer.deployed()`

Reply:
Great catch